libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -41,7 +52,8 @@ describe('ModalHelper', () => {
 
   const spectator = createService({
     service: ModalHelper,
-    imports: [IonicModule.forRoot({ mode: 'ios', _testing: true })],
+    imports: [IonicModule.forRoot({ mode: 'ios', _testing: true }), RouterTestingModule],
+    providers: [mockProvider(ModalOutlet), mockProvider(ChildrenOutletContexts)],
```

Comment:
Consider using `mocks` instead:
```suggestion
    mocks: [ModalOutlet, ChildrenOutletContexts],
```